src/core/ddsc/include/dds/dds.h

```diff
@@ -39,6 +39,7 @@ typedef int32_t dds_entity_t;
 #include "dds/ddsrt/time.h"
 #include "dds/ddsrt/retcode.h"
 #include "dds/ddsrt/log.h"
+#include "dds/ddsrt/misc.h"
```

Comment:
Probably for declaration of `main`. I'm almost certain that that isn't what we want to do